models/intel/noise-suppression-poconetlike-0001/README.md

```diff
@@ -0,0 +1,262 @@
+# noise-suppression-poconetlike-0001
+
+## Use Case and High-Level Description
+
+This is a PoCoNet like model for noise suppression to make speech cleaner.
+The model is based on [PoCoNet](https://arxiv.org/abs/2008.04470) architecure and trained on clean and noise samples from DNS-chalange dataset [github](https://github.com/microsoft/DNS-Challenge/blob/master/README.md) [paper](https://arxiv.org/abs/2101.01902).
+The network works with mono audio sampled on 16kHz.
+The audio processed iterative by patches with 2048 size.
+On each iteration it takes 2048 (128ms) samples as input and returns 2048 (128ms) samples as output with 640 (40ms) samples delay.
+In addition the network required 52 state tensors to make processing seamless.
+On the first iteration these state tensors have to be filled by 0.
+On the consequences iterations theses tensors have to be taken from corresponding outputs of previous iteration.
+You can try [demo](../../../demos/noise_suppression_demo/python/README.md) to see how it works.
+
+## Specification
+
+to process 2048 samples that is 128ms for 16kHz
+
+| Metric            | Value                 |
+|-------------------|-----------------------|
+| GOps              | 1.2                   |
+| MParams           | 7.22                  |
+| Source framework  | PyTorch\*             |
+
+
+
+## Accuracy
+
+The [SISDR](https://arxiv.org/abs/1811.02508) quality metric was calculated on the 100 dev test synthetic speech clips from DNS-chalange dataset [github](https://github.com/microsoft/DNS-Challenge/tree/icassp2021-final/datasets/ICASSP_dev_test_set/track_1/synthetic).
+
+
+| Metric                    | Value         |
+|---------------------------|---------------|
+| SISDR OUT                 |    20.23   dB |
+| SISDR DIFF                |     8.49   dB |
+
+
+## Input
+
+Sequence patch, name: `input`, shape: `1, 2048`, format: `B, T`
+where:
+   - B - batch size
+   - T - number of samples in patch
+
+input state, name: `inp_state_000`, shape: `1, 128`
```

Comment:
do we need detailed specification for all state inputs/output?
Possibly  just provide general info (e.g. inp_state_* - input state layers, should be filled by corresponding out_state from previous step)

Reply:
definitely no. I believed that this is a strong requirement to describe all inputs with sizes. will change it